src/s2i/bin/assemble

```diff
@@ -2,9 +2,11 @@
 
 set -e
 
-echo "---> Copying varnish configuration files..."
+shopt -s dotglob
```

Comment:
Thanks for the PR.

Why is it needed to set this shell option here?

Reply:
As you can see we use it across all containers like here: https://github.com/sclorg/s2i-perl-container/blob/master/5.32/s2i/bin/assemble#L5 https://github.com/sclorg/s2i-python-container/blob/master/src/s2i/bin/assemble#L62. What about in case of dir container '.' files? It will not be moved at all.

Reply:
Yeah, like .git in case user has its own repository. We did it across all containers.
E.g. User has his application in a github repository and does not want to remove all 'all dot files' ,

There are some other files, like Gems, NPMs, and the others.

Reply:
Thanks for the explanation.